[IMP] Maintenance: update add new equipment (#9790)
odoo/documentation: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst

Add maintenance details
-----------------------

The :guilabel:`Maintenance` tab at the bottom of the page provides information about the failure
frequency of the piece of equipment:

- :guilabel:`Expected Mean Time Between Failure`: the average number of days the equipment is
  expected to operate between failures. This number can be configured manually.

> - :guilabel:`Mean Time Between Failure`: the average number of days the equipment operates for

Reviewer: ```suggestion
- :guilabel:`Mean Time Between Failure`: the average number of days the equipment operates
```

  failures. This number is calculated automatically based on previous failures, and cannot
  be configured manually.

> - :guilabel:`Estimated Next Failure`: the estimated date the equipment experiences its next failure.

Reviewer 2: ```suggestion
- :guilabel:`Estimated Next Failure`: the estimated date the equipment may experience its next failure.
```

> - :guilabel:`Estimated Next Failure`: the estimated date on which the equipment experiences its next

Reviewer: ```suggestion
- :guilabel:`Estimated Next Failure`: the estimated date the equipment experiences its next
```

  failure. This date is calculated automatically based on the data in the :guilabel:`Mean Time
  Between Failure` and :guilabel:`Latest Failure` fields, and cannot be configured manually.
- :guilabel:`Latest Failure`: the most recent date on which the equipment failed. This date is based
  on the creation date of the equipment's most recent maintenance request, and cannot be configured
  manually.

> - :guilabel:`Mean Time To Repair`: the average number of days it takes to repair the equipment. This

Reviewer: ```suggestion
- :guilabel:`Mean Time To Repair`: the average number of days needed to repair the equipment. This
```

  number is calculated automatically based on the duration of previous maintenance requests, and
  cannot be configured manually.

.. image:: add_new_equipment/new-equipment-maintenance.png
   :align: center
   :alt: The maintenance tab for a piece of equipment.

.. tip::

>     To see any open maintenance requests for a piece of equipment, go to the page for the equipment
>     and click the :guilabel:`Maintenance` smart button at the top of the page.

Reviewer 2: ```suggestion
    To see any open maintenance requests for a piece of equipment, go to the page for the equipment,
    and click the :guilabel:`Maintenance` smart button at the top of the page.
```
OPTIONAL: added comma for enhanced sentence pacing

>     and select :guilabel:`Maintenance` smart button at the top of the page.

Reviewer: ```suggestion
    and click the :guilabel:`Maintenance` smart button at the top of the page.
```
